src/LorisInstance.php

```diff
@@ -0,0 +1,107 @@
+<?php declare(strict_types=1);
+
+namespace LORIS;
+
+/**
+ * A LorisInstance represents an installed instance of a LORIS
+ * project.
+ *
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ */
+class LorisInstance
+{
+    protected $modulesDirs;
+    private $DB;
+
+    /**
+     * Construct a LORISInstance for the install connected to $db
+     * which uses modules from $moduleDirs.
+     *
+     * @param \Database $db         A database connection to this
+     *                              instance.
+     * @param string[]  $moduleDirs A list of directories that may
```

Comment:
It might be nice to use SPLFileInfo instead especially if the main purpose of this property is to do validation based on the filepaths.

Reply:
That is not the main purpose. The main purpose is to have a list of directories that modules may be found in.